fixtures/vuejs-css-modules/App.vue

```diff
@@ -30,33 +59,3 @@
     display: block;
   }
 </style>
-
-
-<style module="$css">
-  .italic {
-    font-style: italic;
-  }
-</style>
-
-<style lang="scss" module="$scss">
-  .bold {
-    font-weight: bold;
-  }
-</style>
-
-<style lang="less" module="$less">
-  .underline {
-    text-decoration: underline;
-  }
-</style>
-
-<style lang="styl" module="$stylus">
-  .rtl
-    direction: rtl;
-</style>
-
-<style lang="postcss" module="$postcss">
-  .hidden {
-    visibility: hidden;
-  }
-</style>
```

Comment:
The re-ordering works around a vue-loader bug - which was just merged :) https://github.com/vuejs/vue-loader/pull/1671